app/services/call_creator.rb

```diff
@@ -46,10 +46,16 @@ def errors
   private
 
   def sanitize_params!
-    if @params[:member_phone_number].present?
-      @params[:member_phone_number] = Phony.normalize(@params[:member_phone_number])
+    begin
+      if @params[:member_phone_number].present?
+        @params[:member_phone_number] = Phony.normalize(@params[:member_phone_number])
+      end
+    rescue Phony::NormalizationError
+    end
+
+    if @params[:member_id].blank? && @extra_params[:akid].present?
+      @params[:member_id] = Member.find_by_akid(@extra_params[:akid])&.id
```

Comment:
`recognized_member` expects the akit to be at the  root of `params`. And for call actions all tracking params are in `params[:tracking_params]`.